src/web_app/css/main.css

```diff
@@ -104,6 +104,22 @@ footer > div.active {
   opacity: .8;
 }
 
+div#privacy {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  /* Makes sure to stay behind room links so they can be clicked */
+  z-index: -1
```

Comment:
Might be able to get rid of the following footer > div rule instead (but check it doesn't break div#info-div, etc.):

  width: calc(100% - 20px); 

...then add float:left to the div#sharing-div rules, and add the following:

div#privacy {
  display: block;
  float: right;
  opacity: 1;
}

footer {
    position: absolute;
    bottom: 0;
    width: calc(100% - 20px);
    background: black;
    padding: 10px;
    opacity: .8;
}

(...or something like that!)